gnat2goto/driver/tree_walk.adb

```diff
@@ -2135,6 +2300,51 @@ package body Tree_Walk is
             New_Expr : Irep;
          begin
             while Present (Component_Iter) loop
+               if Nkind (Component_Iter) /= N_Allocator
```

Comment:
This would be better written as 

```
not  (Nkind (Component_Iter) in N_Allocator
  | N_Aspect_Specification
  | N_Assignment_Statement
  --   ...
  | N_Unchecked_Type_Conversion)
```